File: py110/small_problems_110_119/easy5/7.py
# https://launchschool.com/exercises/ae172c02

def sum_digits(integer):
    return sum([int(num) for num in str(integer)])

# print(sum_digits(23) == 5)              # True
# print(sum_digits(496) == 19)            # True
# print(sum_digits(123456789) == 45)      # True

File: js210/simpleProblems/easy3/7.js
// https://launchschool.com/exercises/f3a0ccc0

// function runningTotal(array) {
//   let result = [];
//   let total = 0;

//   array.forEach(num => {
//     total += num;
//     result.push(total);
//   });

//   return result;
// }

function runningTotal(array) {
  let total = 0;

  return array.map(num => {
    total += num;
    return total;
  });
}

runningTotal([2, 5, 13]);             // [2, 7, 20]
runningTotal([14, 11, 7, 15, 20]);    // [14, 25, 32, 47, 67]
runningTotal([3]);                    // [3]
runningTotal([]);                     // []